ow_lander/src/ow_lander/mixins.py

```diff
@@ -130,63 +130,52 @@ def execute_action(self, goal):
   def plan_trajectory(self, goal):
     pass
 
-class ModifyPoseMixin:
+
+class FrameMixin:
   """Can be inherited by ArmMoveActions that modify pose. DOES NOT implement an
   arm interface, and so must be inherited along with ArmActionMixin or one of
   its children.
   """
 
-  ARM_END_EFFECTOR = 'l_scoop_tip'
   COMPARISON_FRAME = 'world'
+  END_EFFECTOR = 'l_scoop_tip'
+  END_EFFECTOR_FRAME = 'base_link'
```

Comment:
This constant's name feels wrong because base_link is not an end effector. Is there a more appropriate name that can be used?

Reply:
Its meant to denote "the frame of the end effector", which is the frame MoveGroupCommander will use to pose the arm by default. Let me rethink the name. Alternatively, I may enforce better encapsulation since its really a ArmTrajectoryPlanner detail, not something actions should have to worry about.

Reply:
The original names seem clear to me, actually.

Reply:
In either case I redid a function so that `END_EFFECTOR_FRAME` is no longer needed as a class variable, and I think its cleaner.